osu.Game/Skinning/Editor/SkinSelectionHandler.cs

```diff
@@ -281,24 +281,21 @@ private static Anchor getClosestAnchor(Drawable drawable)
             if (parent == null)
                 return drawable.Anchor;
 
-            static Vector2 getOriginPositionFromQuad(in Quad quad, Anchor origin)
+            var screenPosition = drawable.ScreenSpaceDrawQuad.TopLeft;
             {
```

Comment:
Do we use scope braces here?

Reply:
Scope braces are rarely used around these parts. You're probably better off splitting a (local?) method.